### DISCUSSION TITLE
How could I change font and font size?

### DISCUSSION THREAD

**Discussion by yoonghm** [PgBiel/typst-tablex#89] [Q&A]
I checked this repository, I could not find a way to change font and font size. Could you advise? Thanks.

**Comment by PgBiel** [comment]
Hello, you may use Typst's `set` rules (not only for tablex, but in any other context). Please see here for more information: https://typst.app/docs/reference/styling/

For instance:
```typ
#set text(font: "New Computer Modern")
#set text(20pt)
```

**Reply by yoonghm** [reply]
I figured out using this:

```typst
#figure(
  tablex(
    columns: 2,
    align: center + horizon,
    auto-vlines: false,
    header-rows: 1,
    map-cells: cell => {
      if cell.y == 0 {
        set text(size: 14pt)
        strong(cell.content)
      }
      else {
        set text(size: 12pt)
        cell.content
      }
    },
  
    /* Header */
    [Index], [Example],
    [`A`  ], [Apple],
    [`B`  ], [Ball],
    [`C`  ], [Car],
  ),
  kind: table,
  caption: "English alphabet"
)
```

![image](https://github.com/PgBiel/typst-tablex/assets/4215991/f5eafd98-a204-4c17-9775-c4db976ebc41)

Thanks!
